src/main/scala/temple/generate/target/openapi/OpenAPIGenerator.scala

```diff
@@ -37,18 +38,28 @@ private class OpenAPIGenerator private (
 
   private val paths = mutable.LinkedHashMap[String, Path.Mutable]()
 
-  private def path(url: String): mutable.Map[HTTPVerb, Handler] =
-    paths.getOrElseUpdate(url, Path.Mutable()).handlers
+  private def path(url: String, lowerName: String): mutable.Map[HTTPVerb, Handler] = {
+    val idParameter = when(url.contains("{id}")) {
+      Parameter(
+        InPath,
+        name = "id",
+        required = Some(true),
+        schema = OpenAPISimpleType("string", "uuid"),
+        description = s"ID of the $lowerName to perform operations on",
+      )
+    }
 
-  private def pathWithID(url: String, lowerName: String): mutable.Map[HTTPVerb, Handler] = {
-    val parameter = Parameter(
-      InPath,
-      name = "id",
-      required = Some(true),
-      schema = OpenAPISimpleType("number", "int32"),
-      description = s"ID of the $lowerName to perform operations on",
-    )
-    paths.getOrElseUpdate(url, Path.Mutable(parameters = Seq(parameter))).handlers
+    val parentIDParameter = when(url.contains("{parent_id}")) {
+      Parameter(
+        InPath,
+        name = "parent_id",
+        required = Some(true),
+        schema = OpenAPISimpleType("string", "uuid"),
+        description = s"ID of the parent entity to perform operations on",
```

Comment:
not sure about this description

Reply:
What would you suggest?

Reply:
"ID of the parent entity of the entity to perform operations on"?